### ISSUE TITLE
"looking for the kun'yomi reading" pop-up obscures "view the correct reading" button

### ISSUE THREAD

**Issue by elbie** [xiprox/WaniKani-for-Android#31]
If you enter the wrong reading for a kanji, the "WaniKani is looking for the kun'yomi reading" pop-up will appear, but not disappear.  If you then enter the incorrect answer, the "Need help? View the correct reading and mnemonic" pop-up will appear, but the "looking for the kun'yomi" pop-up prevents you from actually clicking on the correct reading button.

![screenshot_20161103-080508](https://cloud.githubusercontent.com/assets/1656917/20649609/69751a9c-b491-11e6-9b13-cb7cbf5b375e.png)

**Comment by earthiverse** [comment]
I have this problem as well, but this bug can't be solved by this application.
It's a problem with WaniKani itself, as this is the same page you get when you visit WaniKani on the computer.

![image](https://cloud.githubusercontent.com/assets/602905/21598343/9a210aa0-d19e-11e6-9a87-b9f62cfd479f.png)
But it actually works in Chrome, lol. I think there's a fade animation and it doesn't trigger on mobile for some reason...
![image](https://cloud.githubusercontent.com/assets/602905/21598345/a63aaff8-d19e-11e6-9f2d-d3383a059ae6.png)

**Comment by xiprox** [comment]
Yeah, I thought about that being the case at first but browsers seem to work fine. The thing is, we have quite a bit of customization in our browser implementation. Will have to look into it sometime.
